src/index.js

```diff
@@ -67,7 +67,7 @@ function applyRef (e, ref) {
 function ensureAttrs (objs) {
   const { attrs, events, ref, key, ...props } = objs || {};
   const newRef = ensureRef({ attrs, events, props, ref });
-  return { ref: newRef, ...(key ? { key } : {}) }
```

Comment:
I wonder if it's okay just to pass `key`, even if it's undefined? We wouldn't need any checks if that's the case. WDYT?